schemas/cloudToDevice/ground_fix/ground-fix-example-with-meta.json

```diff
@@ -0,0 +1,15 @@
+{
+    "appId": "GROUND_FIX",
+    "messageType": "DATA",
+    "data": {
+        "lat": -45,
+        "lon": 100,
+        "uncertainty": 500,
+        "fulfilledWith": "WIFI",
+        "meta": {
+            "anchors": [{ 
```

Comment:
is having `"fulfilledWith": "ANCHOR"` necessary?
wouldn't the presence of an `"anchors"` object be sufficient?

Reply:
I went back and forth on this one. Ultimately we don't need it.

Reply:
did you decide to keep this?

Reply:
This is outdated.